### ISSUE TITLE
Poor image quality even with 8M camera

### ISSUE THREAD

**Issue by bratenn** [jhuckaby/webcamjs#51]
Hello,  I am using a Dell Venue Windows 8.1 Pro 64-bit with an 8 Megapixel back camera.  When using the camera app on the Venue, it takes a very high quality photo.   When using webcam.js, the preview is very wavy and grainy, and the final snapshot is as well.  I've tried different settings in "Webcam.set", but still poor image quality even when I attempt to match the resolution of the camera app.  Any ideas?  I've attached two files....one is a screen shot of the webcamjs, the other is an photo taken in the Venue camera app.  You can see a lot more detail in the camera app photo.

![webcamjs](https://cloud.githubusercontent.com/assets/10746191/5944859/320059d0-a6f8-11e4-8a15-de1442a59140.png)
![venue_camera_app](https://cloud.githubusercontent.com/assets/10746191/5944861/32483e08-a6f8-11e4-8976-429542619142.JPG)

**Comment by jhuckaby** [comment]
Yeah, I've had problems with HD images as well.  It seems the driver support in both Flash and HTML5 GetUserMedia is very poor for images greater than 640x480.  Chrome seems to warp / stretch the image, and other browsers deliver blurry / scaled results.  I've only gotten it to work with certain cameras in certain browsers.  I'm sorry you ran into this.  I will add a disclaimer to the README.

**Comment by bratenn** [comment]
OK, Thanks for the feedback!

**Comment by jhuckaby** [comment]
If you can, please try this again with the latest library (v1.0.1), as I added better HD settings in getUserMedia.  Thanks!

http://pixlcore.com/demos/webcamjs/demos/hd.html

**Comment by simonmysun** [comment]
It seems that I have to specify my camera solution?

**Comment by mikhwana1** [comment]
i have  tried the latest library (v1.0.1) for better hd settings
but it didnt work and still poor image quality 
can you help me to complete our issue about it

**Comment by mikhwana1** [comment]
`

Webcam.set({
--
  | width: 1280,
  | height: 720,
  | image_format: 'jpeg',
  | jpeg_quality: 90,
  | constraints: {
  | width: { exact: 1280 },
  | height: { exact: 720 }
  | }
  | });
  | Webcam.attach( '#my_camera' );
  | </script>

`
that's the latest library (v1.01) for better hd settings right?
but still didn't work
